src/ape/managers/chain.py

```diff
@@ -1111,7 +1115,10 @@ def instance_at(
                 raise  # Current exception
 
         if not contract_type:
-            raise ChainError(f"Failed to get contract type for address '{contract_address}'.")
+            msg = f"Failed to get contract type for address '{contract_address}'."
+            if self.provider.network.explorer is None:
+                msg += " Try installing an explorer plugin, such as Etherscan."
```

Comment:
(notes from offline)

- [ ] handle case where using provider without explorer (mention also try using diff network)
- [ ] mention cmd to install plugin
- [ ] if all seems good here, mention that the contract may not be verified